packages/verification-vue-component/src/locales/de.json

```diff
@@ -69,6 +70,7 @@
                 "registrationDate": "Ausstelldatum & -zeit",
                 "registrationTransaction": "Ausstellungstransaktion",
                 "revocationTransaction": "Annullierungstransaktion",
+                "retractionTransaction": "Rückzugstransaktion",
                 "revocationDate": "Annullierungsdatum & -zeit",
```

Comment:
@alnicoli Es gibt hier ein "revocationDate". Ein "retractionDate" gibt es nicht?

Reply:
Dieses wird nur bei der certify in der Expertinfo verwendet, bei sign zeigen wir das revocationDate nicht an bzw. wird diese Translation nicht verwendet. Daher habe ich für das retractionDate gar nicht erst ein Label erstellt.